app/views/sidebar/_dashboard.html.erb

```diff
@@ -18,13 +18,20 @@
   </div>
   
 <% if current_user %>
-  <div style="margin: 2.2rem 0;">
-    <a class="btn btn-lg btn-outline-secondary blurred-location-input" submit_to="/profile/tags/create/<%= current_user.uid %>"><i class="fa fa-map-marker" style="color:#c40;"></i> Add your location</a>
-    <p style="font-size: smaller; margin: 0.3rem 0;">
-      Add your location to see work near you<br>
-      <a href="https://publiclab.org/location-privacy" target="_blank">About location privacy »</a>
-    </p>
-  </div>
+  <% if current_user.has_power_tag("lat") && current_user.has_power_tag("lon") && current_user.has_power_tag("zoom") %>
```

Comment:
hi @gauravahlawat81 this looks good.. in some cases the `"zoom"` is optional, it might not be present at all times.
```suggestion
  <% if current_user.has_power_tag("lat") && current_user.has_power_tag("lon")%>
```

Reply:
Thanks for the super helpful feedback @RuthNjeri :smile: